DEPLOYNOTES.rst

```diff
@@ -7,14 +7,29 @@ Deploy and Upgrade notes
 ----
 
 * Switching from Mezzanine to Wagtail requires a manual migration *before*
-  installing the new version to avoid migration dependency conflicts:
+  installing the new version to avoid migration dependency conflicts::
 
      python manage.py migrate pages zero
 
 * Wagtail provides predefined groups for *Editor* and *Moderator*. Users
   who were previously in the *Content Editor* group should be added
   to one of these, and the *Content Editor* group should be removed.
 
+ * Solr schema changes are required for this update as well as a reindex of
+   both works and pages. Before these can be run, however, an updated
+   ``solrconfig.xml`` that includes the appropriate ``<lib/>`` declarations
+   must be installed on the core. A sample of the correct declarations can
+   be found in ``ci/solrconfig.xml``. The following lines must be added::
```

Comment:
Are we not providing a `solrconfig.xml` ? Or were we just using defaults before now?

Seems easier to me to provide a local version vs documenting what to add manually.

Reply:
The `solrconfig.xml` could be used and copied over. However, I am reluctant to make the instruction to copy it over because of other defaults or changes that might be affected on the production core's `solrconfig' vs providing the `<lib/>` lines that can be added.

Reply:
Would it be better to copy over what production's config looks like and make it our template going forward (perhaps as `.xml.sample` somewhere logical (`archive/fixtures` but seems off)?

Reply:
@bwhicks that sounds like a good solution to me. Maybe a top-level solr or solr_conf directory, something like we did for Winthrop? We might eventually want to customize other things too.